tests/scripts_test_result.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
```

Comment:
please explain how to run this file and how what the file does in a comment at the beginning of the file.

Reply:
also please explain how to run it in the readme